gqlauth/core/middlewares.py

```diff
@@ -112,7 +112,7 @@ def _inject_user_and_errors(kwargs: dict) -> UserOrError:
         # channels compat
         if isinstance(context, dict):
             request = context["request"]
-            user_or_error: UserOrError = request.scope[USER_OR_ERROR_KEY]
+            user_or_error: UserOrError = request.consumer.scope[USER_OR_ERROR_KEY]
```

Comment:
```suggestion
            try:
            	user_or_error: UserOrError = request.consumer.scope[USER_OR_ERROR_KEY]
            except AttributeError:
                # FIXME: This is here because the test consumer is not 1:1 as a real client ATM
            	user_or_error = request.scope[USER_OR_ERROR_KEY]
```